### PR TITLE
Remove, comment of fix unreachable code

### PR DESCRIPTION
**Pull request by joren485**
##### SUMMARY

There is some code that cannot be reached.

This PR removes code that cannot be reached (Please tell me if code should be commented instead of removed).  

If there is  a `FIXME` in the comments, this PR comments the unreachable code (Please tell me if that code should just be removed).  

And in one case, this PR fixes an unreachable `else` statement.

##### ISSUE TYPE

 - Bugfix Pull Request

### CODE REVIEW

**Comment by joren485** [comment]
ready_for_review

**Comment by willthames** [comment]
I'm fine with the SSM module change but would be easier to take that as a separate PR.

**Comment by joren485** [comment]
Splitting this in separate PRs

File: lib/ansible/cli/config.py [hunk 1]
```diff
@@ -133,14 +115,6 @@ def execute_edit(self):
         '''
         raise AnsibleError("Option not implemented yet")
 
-        # pylint: disable=unreachable
```
Comment (bcoca) [inline lib/ansible/cli/config.py:136]: these are placeholders as hints of what i need to do when implementing the features, see 'pylint hint'

**Pull request closed without merging**